AppCUI/src/OS/Windows/File.cpp

```diff
@@ -6,23 +6,28 @@ using namespace OS;
 using namespace Utils;
 
 #define VALIDATE_FILE_HANLDE(returnValue)                                                                              \
-    CHECK(this->FileID.Handle != INVALID_HANDLE_VALUE, returnValue, "File has not been opened !");
+    CHECK(this->FileID.Handle != INVALID_HANDLE_VALUE, returnValue, "File has not been opened!");
 #define F_HNDL ((HANDLE) this->FileID.Handle)
 
+static const std::u16string longPathPrefix{ uR"(\\?\)" };
+
 File::File()
 {
-    this->FileID.Handle = INVALID_HANDLE_VALUE;
+    FileID.Handle = INVALID_HANDLE_VALUE;
 }
+
 File::~File()
 {
-    this->Close();
+    Close();
 }
 
 bool File::OpenWrite(const std::filesystem::path& filePath)
 {
     Close();
-    HANDLE hFile = CreateFileW(
-          (LPCWSTR) filePath.u16string().c_str(),
+
```

Comment:
As sugera un alt approach (ca sa evitam inca o alocare in plus).
`LocalUnicodeStringBuilder<1024> temp;
temp.Add(longPathPrefix);
temp.Add(path.u16string());
`
Also ... ar fi util sa verificam daca nu incepe deja cu acel prefix !

Reply:
done